src/Data/Array/Destination.hs

```diff
@@ -70,22 +148,21 @@ fill = Unsafe.toLinear2 unsafeFill
 
 -- | @'split' n dest = (destl, destr)@ such as @destl@ has length @n@.
 --
--- 'split' is total: if @n@ is larger than the length of @dest@, then @destr@ is
--- empty.
+-- 'split' is total: if @n@ is larger than the length of @dest@, then
+-- @destr@ is empty.
 split :: Int -> DArray a #-> (DArray a, DArray a)
 split n = Unsafe.toLinear unsafeSplit
   where
     unsafeSplit (DArray ds) =
       let (dsl, dsr) = MVector.splitAt n ds in
         (DArray dsl, DArray dsr)
 
--- | Convert a Vector into a destination array to be filled, possibly with
--- a conversion function.
--- Assumes both arrays have the same size.
+-- | Convert a Vector into a destination array to be filled, possibly with a
+-- conversion function.  Assumes both arrays have the same size.
```

Comment:
Wouldn't it be more straightforward to say something like:

> Fills the destination array with the contents of given vector

Because, to me, the result is not "a destination array to be filled".

Reply:
Agreed.